gymnasium_robotics/envs/multiagent_mujoco/coupled_half_cheetah.py

```diff
@@ -0,0 +1,116 @@
+import os
+
+import gymnasium
+import numpy as np
+from gymnasium.envs.mujoco import mujoco_env
+from gymnasium.utils.ezpickle import EzPickle
+
+DEFAULT_CAMERA_CONFIG = {
+    "distance": 4.0,
+}
+
+
+class CoupledHalfCheetah(mujoco_env.MujocoEnv, EzPickle):
```

Comment:
Would we want to register these environments like the other mujoco envs?

Reply:
I have been Thinking about this, That can be considered after `MaMuJuCo` is finished and `MuJoCo` environments have moved to `Gymnasium-Robotics` (potentially with added with `MuJoCo-v5`)